src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -564,6 +564,10 @@ Either change the value in the AndroidManifest.xml to match the $(SupportedOSPla
     <value>This could be caused by the project having non-ASCII characters in its filename or path. See https://learn.microsoft.com/dotnet/android/messages/apt2265 for details.</value>
     <comment>The following are literal names and should not be translated:</comment>
   </data>
+  <data name="APT2266" xml:space="preserve">
+    <value>Asset directories cannot use non ascii characters. This could be caused by the directory having non-ASCII characters in its filename or path. See https://learn.microsoft.com/dotnet/android/messages/apt2266 for details.</value>
```

Comment:
Should it be:
```suggestion
    <value>Asset directories cannot use non-ASCII characters. This could be caused by the directory having non-ASCII characters in its filename or path. See https://learn.microsoft.com/dotnet/android/messages/apt2266 for details.</value>
```